RockWeb/Blocks/Groups/GroupAttendanceDetail.ascx.cs

```diff
@@ -409,14 +425,9 @@ protected void lvPendingMembers_ItemCommand( object sender, ListViewCommandEvent
         /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
         protected void lbAddMember_Click( object sender, EventArgs e )
         {
-            this.IsEditingGroupMember = true;
             var personAddPage = GetAttributeValue( "GroupMemberAddPage" );
 
-            if ( personAddPage == null || personAddPage == string.Empty )
-            {
-                DisplayEditGroupMember( 0 );
-            }
-            else
+            if ( !personAddPage.IsNullOrWhiteSpace() )
             {
                 // redirect to the add page provided
                 if ( _group != null && _occurrence != null )
```

Comment:
Nice catch in here -- just below.  (You save the attendance before you navigate away.)  Sharp eye for detail and edge cases!